### PR TITLE
[WDFT-AMS-1020] Yann Quintin

### PR DESCRIPTION
**Pull request by YannQuintin**
The first 6 iterations are in the box although all points on Jasmin are not showing green.

All green: 1, 2, 5
Partially green: 3, 4.
All red: 6.

Iteration 6: wrote this one with Malcom and yet no point turn green on this one. Will go through it again after reviewing the lesson + doing some classes on freecodecamp.

### CODE REVIEW

File: src/movies.js [hunk 1]
```diff
@@ -1,16 +1,97 @@
...
+    const dramaArr = movies.filter(
+      films => films.genre.includes("Drama")
+    )  
+    const dramaRates = dramaArr.map(films => films.rate)
+    console.log(dramaRates)
+    const sum = dramaRates.reduce((totalRate, singleRate) => totalRate + singleRate)
+    return +(sum / dramaRates.length).toFixed(2)
+    console.log(sum)
+  }
+
```
Comment (LloydC) [inline src/movies.js:62]: You can remove line 56-60 if you make use of function `ratesAverage`

`return ratesAverage(dramaArr)`

File: src/movies.js [hunk 2]
```diff
@@ -1,16 +1,97 @@
...
+        return 1
+      } else if (b.title > a.title) {
+        return -1
+      } else {
+        return 0
+      }
+    let moviesTitleList = movies.map(
+      items => items.title)
+    })
+    console.log(moviesTitleList)
```
Comment (LloydC) [inline src/movies.js:90]: You should always have a `return` statement at the end of your function to store the returned value. Additionally you were meant to print the first 20 titles.

`return moviesTitleList.slice(0, 20)`

Comment (YannQuintin) [reply src/movies.js:90]: Thx, Added the returns.

**Comment by YannQuintin** [comment]
Hey TAs

This is the latest commit. I've done as many iterations from 1 through to 6. 

Jasmine counts 33 failures, even on some iterations I wrote. I guess my solutions do not match the tests?

I've got quite some work to do on Javascript basics over the weekend as I am still struggling.

**Pull request closed without merging**